database.sql

```diff
@@ -4,23 +4,40 @@ USE groupc_kainos_recruitment_JenniferC;
 CREATE TABLE IF NOT EXISTS job_roles (
 id smallint PRIMARY KEY AUTO_INCREMENT,
 name varchar(50),
-job_description varchar(500)
+job_description varchar(500),
+band_id smallint,
+FOREIGN KEY (band_id) REFERENCES band_levels(band_id)
+);
+
+CREATE TABLE IF NOT EXISTS band_levels (
+band_id smallint PRIMARY KEY AUTO_INCREMENT,
+band_name varchar(25)
```

Comment:
As with capability, band is from a set of specific values so how about using enum data type?

Reply:
Not really sure what is meant here.